src/sentry/static/sentry/app/views/dashboardsV2/detail.tsx

```diff
@@ -36,32 +38,19 @@ type Props = {
 
 type State = {
   dashboardState: DashboardState;
-  changesDashboard: DashboardListItem | undefined;
+  changesDashboard: DashboardDetails | null;
```

Comment:
This `changesDashboard` is a little unclear, maybe? isn't it just `dashboardData`? Since it's a union of a prebuilt and a real one? Maybe just the verb `changes` was tripping me up.

Reply:
I'm not a fan of the name either. I'll change it in a follow up pull request.